phone_bank_result.md

```diff
@@ -0,0 +1,804 @@
+
+A phonebank result occurs when one person attempts to contact another person in a phonebank.  For example, a campaign staffer calls a volunteer to ask whether he intends to attend the upcoming kick-off event.  The person being contacted is known as the "target" of the phonebank, and the target may have [Answers](answers.html) or [Taggings](taggings.html) applied to them as a result.  Phonebank result is always part of a [Phonebank](phone_bank.html).
+
+### Sections
+
+* [Endpoints and URL structures](#endpoints-and-url-structures)
+* [Fields](#fields)
+    * [Common Fields](#common-fields)
+    * [Canvass Fields](#phone-bank-result-fields)
+    * [Links](#links)
+* [Related Resources](#related-resources)
+* [Scenarios](#scenarios)
+    * [Scenario: Retrieving a collection of Phonebank result resources for a person (GET)](#scenario-retrieving-a-collection-of-phone-bank-result-for-a-person-get)
+    * [Scenario: Retrieving a collection of Phonebank result resources for a phone bank (GET)](#scenario-retrieving-a-collection-of-phone-bank-result-for-a-phone-bank-get)
+    * [Scenario: Retrieving an individual Phonebank result resource (GET)](#scenario-retrieving-an-individual-phone-bank-result-get)
+    * [Scenario: Retrieving Answers for an individual Phonebank result resource (GET)](#scenario-retrieving-answers-for-an-individual-phone-bank-result-get)
+    * [Scenario: Retrieving Taggings for an individual Phonebank result resource (GET)](#scenario-retrieving-taggings-for-an-individual-phone-bank-result-get)
+    * [Scenario: Creating a new Phonebank result (POST)](#scenario-creating-a-new-phone-bank-result-post)
+    * [Scenario: Modifying a Phonebank result (PUT)](#scenario-modifying-a-phone-bank-result-put)
+    * [Scenario: Deleting a Phonebank result (DELETE)](#scenario-deleting-a-phone-bank-result-delete)
+
+
+{% include endpoints_and_url_structures.md %}
+
+The link relation label for an Phone bank result resource is ```osdi:phone-bank-result``` for a single resource or ```osdi:phone-bank-results``` for a collection of resources.
+
+_[Back to top...](#)_
+
+
+## Fields
+
+{% include fields_intro.md %}
+
+{% include global_fields.md %}
+
+_[Back to top...](#)_
+
+
+### Phone bank result Fields
+
+| Name          | Type      | Description
+|-----------    |-----------|--------------
+|origin_system	  |string    |A human readable identifier of the system where this Phonebank result was created. (ex: "OSDI System")
+|call_date		  |string	 |The date and time the call was made.
+|caller_id        |string    |The phone number that is displayed as the caller-id for this call
+|phone_number     |string    |The phone number that was called
+|duration         |string    |The length of the phone call in seconds. Calls that do not connect will have zero in this field.
+|disposition      |enum      |Disposition of the phone call. Possible values are "answer". "busy", "do-not-call", "meaningful interaction", "machine" etc..
```

Comment:
probably should be called status_code, see canvasses, and maybe add the success bool too?